compose/reference/build.md

```diff
@@ -22,7 +22,7 @@ Services are built once and then tagged, by default as `project_service`. For
 example, `composetest_db`. If the Compose file specifies an
 [image](/compose/compose-file/index.md#image) name, the image is
 tagged with that name, substituting any variables beforehand. See [variable
-substitution](#variable-substitution).
+substitution](compose/compose-file/#variable-substitution).
```

Comment:
For this to work it you need to make it an absolute link like `/compose/...`. As you have it right now, this will try to find a file at `/compose/reference/compose/compose-file`, which doesn't exist.